tsconfig.json

```diff
@@ -11,6 +11,6 @@
     "esModuleInterop": true
   },
   "typeRoots": ["node_modules/@types"],
-  "exclude": ["node_modules", "lib"],
+  "exclude": ["node_modules", "lib", "**/__tests__", "**/__mocks__"],
```

Comment:
This makes `tests` and `mocks` fallback to default TS config, which generates warnings and is quite unpredictable - would rather add this change it test files had their own configuration.

Reply:
I agree, it would make sense to have a `tsconfig.test.json` for the test files; I've excluded the mocks and tests because they were also compiled and were included in the compiled output which isn't very great imo 😅